examples/streams/streams.bal

```diff
@@ -1,70 +1,74 @@
 
-    //Create a stream that is constrained by the StatusCount struct type.
-    stream<StatusCount> filteredStatusCountStream;
+    //Declare a stream constrained by float values.
+    stream<float> temperatureStream;
 
-    //Create a stream that is constrained by the Teacher struct type.
-    stream<Teacher> teacherStream;
+    //Subscribe to the temperature stream with a function accepting float events.
+    temperatureStream.subscribe(printTemperature);
 
-    //Invoke the method which contains the forever streaming statement.
-    testAggregationQuery(filteredStatusCountStream, teacherStream);
+    //Publish float events to the stream, indicating temperature
+    temperatureStream.publish(28.0);
+    temperatureStream.publish(30.1);
+    temperatureStream.publish(29.5);
+
+    //Allow for receipt by subscribers. The printTemperature function would be invoked for each published event.
+    runtime:sleep(1000);
 
-    //Create sample events. These events are sent into the `teacherStream` input stream.
-    Teacher t1 = {name:"Raja", age:25, status:"single", batch:"LK2014", school:"Hindu College"};
-    Teacher t2 = {name:"Shareek", age:33, status:"single", batch:"LK1998", school:"Thomas College"};
-    Teacher t3 = {name:"Nimal", age:45, status:"married", batch:"LK1988", school:"Ananda College"};
 
-    //Subscribe the `filteredStatusCountStream` stream to the `printStatusCount` function. Each time the stream
-    // receives an event, this function is called.
-    filteredStatusCountStream.subscribe(printStatusCount);
+    //Declare an unconstrained stream which would accept events of any type, equivalent to stream<any>.
+    stream updateStream;
 
-    //Publish the events that were generated in the above step.
-    teacherStream.publish(t1);
-    teacherStream.publish(t2);
-    teacherStream.publish(t3);
+    //Subscribe to the unconstrained stream with a function accepting events of any type.
+    updateStream.subscribe(printEvent);
 
+    //Publish events to the stream.
+    updateStream.publish("Hello Ballerina!");
+    updateStream.publish(1.0);
+    updateStream.publish(e1);
+
+    //Allow for receipt by subscribers. The printEvent function would be invoked for each published event.
     runtime:sleep(1000);
+
 }
 
-//Print the output events.
-function printStatusCount (StatusCount s) {
-    io:println("Event received; status: " + s.status +" and total occurrences: "+s.totalCount);
+//Function used to subscribe to a stream accepting employee events.
+function printEmployeeName(Employee employee) {
+    io:println("Employee event received for Employee Name: " + employee.name);
 }
 
+//Function used to subscribe to a stream accepting float events.
+function printTemperature(float temperature) {
+    io:println("Temperature event received: " + temperature);
+}
+
+//Function used to subscribe to a stream accepting events of any type.
+function printEvent(any event) {
+    io:println("Event received: " + <string>event);
```

Comment:
Now, io:println("Event received: ", <string>event); 
should work

Reply:
Fixed in 70258d2